go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/xline-kv/go-xline
+
+go 1.20
```

Comment:
the go version in CI is 1.21, which is somewhat different from here. Can we unify the version?

Reply:
done